doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -425,7 +425,7 @@ features [here](../../../ref_model/chapters/chapter05.md#5.4).
 
 <a name="3.2.2.1"></a>
  ### 3.2.2.1 Kubernetes Networking Semantics
-The support for advanced network configuration management is non existing in Kubernetes proper. Kubernetes is missing advanced networking configuration component of Infrastructure as a Service (IaaS). Compared to OpenStack, there is no network configuration API like Neutron, and there is no way to create L2 networks, instantiate network services such as L3aaS and LBaaS and then a way connect them all together like in case of Neutron.
+The support for advanced network configuration management doesn't exist in core Kubernetes. Kubernetes is missing the advanced networking configuration component of Infrastructure as a Service (IaaS). For example, there is no network configuration API, there is no way to create L2 networks, instantiate network services such as L3aaS and LBaaS and then connect them all together.
```

Comment:
Maybe we should also mention that it is not possible to manage the networks in an explicit way.

Reply:
Yes, I will add in something about that